packages/api/helper/mailer.js

```diff
@@ -17,8 +15,8 @@ const transport = {
   service: 'gmail',
   host: 'smtp.gmail.com',
   auth: {
-    user: process.env.MAILER_USER,
-    pass: process.env.MAILER_PASS,
+    user: process.env.MAILER_USER2,
+    pass: process.env.MAILER_PASS2,
```

Comment:
Can we name these something a little more descriptive, like `MAILER_USER_NOMS`?

Alternatively, if *all* the mail we're sending is coming from `nomination@`, could we just change the values of the existing variables?

Reply:
Yes we can.